### ISSUE TITLE
Suggestion - More strict cut between original and adapted

### ISSUE THREAD

**Issue by team4music** [chrislgarry/Apollo-11#124]
Hi, 

a real great piece of history. But what i don't like is that there are a lot of comments mixed together. It would be geat if its clear to see which is the original code from 1969 (including comments) and which has been made later.

**Comment by chrislgarry** [comment]
@team4music we are working on fixing up the typos from digitization. we can then move on to removing anything that wasn't there to begin with, but I need to make sure this is ok. I agree 100%. It's like going to a museum and seeing annotations on the actual artifacts.

**Comment by badvision** [comment]
I suggest having a historic branch so that any future commits don't affect the original.

**Comment by wopian** [comment]
@badvision the future commits to this repo are fixing the digitised code to fully match the original images.

**Comment by chrislgarry** [comment]
@badvision that's what the master branch is aiming for.

**Comment by RubenKelevra** [comment]
Would be also great of we manage different apollo versions in different branches. Would be nice to see the diffs over time. :)

**Comment by chrislgarry** [comment]
@RubenKelevra another great idea. The source is all there in Virtual AGC. We just need to pull it in. I think we should focus on getting Apollo 11 into a steady state first with most of the issues resolved. Otherwise we will have half-correct versions of all of them.

**Comment by rburkey2005** [comment]
I know this is quite an old topic now, but I want to correct some out-dated info I gave about it earlier.

I believe that Chris once asked me how or if we (at the Virtual AGC project) distinguish between the original comments in the AGC program code and "modern" comments we've added to it.  At the time, I told him that we didn't have any hard-and-fast way of doing it, but that modern comments tended to be mixed upper-and-lower case, while the original comments were all upper-case and had other stylistic differences.

That's not a satisfactory answer, really, and a renewed discussion about that came up over here yesterday, and we've made some changes to come up with what we think is a solution.  As you know, comments are preceded (in our source files, anyway) by a "#" character.  We have also sometimes been using comments preceded by "##" for comments that receive special treatment within our colorized, syntax-highlighted AGC program listings.  The suggestion that came up over here was simply to use ## for all newly-added, modern comments, but continue to use '#' for the original comments.  The distinction is then clear.

In the colorized, syntax-highlighted listings, the distinction between these two is even more obvious, since the ## comments are displayed in fancy boxes with an Apollo graphic, whereas the # comments are simply displayed as normal within the flow of the code.

There are a couple of exceptions to the last paragraph, in that embedded page numbers like "## Page 123", or the file headers containing lots and lots of comments at the top of the file, still would show up as as normal comments (though preceded by ##) in the highlighted text and wouldn't have any fancy formatting.  That's to prevent them from taking over the syntax-highlighted listings.

Anyhow, this is really new, so we haven't really had any chance to put this into effect in the various AGC program listings in our inventory, or the several new ones we're currently in the process of transcribing, but we seem to be of the opinion that it's a pretty good approach ... so I offer it up to you as a possibility.  :-)
